modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -72,10 +72,11 @@ static inline DST sub(SRC1 x, SRC2 y)
     return saturate<DST>(x - y, roundf);
 }
 
+// there is no difference between sub and subr (due to documentation)
```

Comment:
Could you please give me a link to this documents?

Reply:
https://docs.opencv.org/4.x/da/dd3/group__gapi__math.html#:~:text=%E2%97%86%C2%A0-,subRC(),-GMat%20cv%3A%3Agapi

Reply:
I haven't seen such statement in mentioned documentation.

Reply:
I mean the order of arguments. It is the same as it in sub function

Reply:
The difference is obvious. SubC: SRC - C; SubRC: C - SRC.
It's necessary either use `sub()` instead of `subr()` and then remove  `subr()` at all, or roll back these changes and leave everything as it was.

Reply:
I would prefer the second variant.

Reply:
Applied